tasks/Functions-client/setAutoRequest.js

```diff
@@ -60,6 +61,13 @@ const setAutoRequest = async (contract, taskArgs) => {
   )
 
   const store = new RequestStore(hre.network.config.chainId, network.name, "automatedConsumer")
+  const previousSecretURLs = []
+  try {
+    const artifact = await store.read(taskArgs.contract)
+    if (artifact.activeManagedSecretsURLs) previousSecretURLs = artifact.secretsURLs
+  } catch {
+    /* new request, continue */
```

Comment:
Just to clarify, this `catch` should only occur if it is a new _contract_

It would always be a new request